tests/test_cache.py

```diff
@@ -15,3 +21,45 @@ def test_disk_cache(tmpdir):
 
     cache.set("testing", "foo", "baz")
     assert cache.get("testing", "foo") == "baz"
+
+
+def test_get_pkg_unique_identifier(monkeypatch):
+    monkeypatch.setattr(sys, "version_info", (3, 8, 1, "final", 0))
+    monkeypatch.setattr(sys, "prefix", "/home/john/.pyenv/versions/myvirtualenv")
+
+    mock_version_module = types.ModuleType('tldextract._version', 'mocked module')
+    mock_version_module.version = "1.2.3"
+    monkeypatch.setitem(sys.modules, "tldextract._version", mock_version_module)
+
+    assert get_pkg_unique_identifier() == "3.8.1.final__myvirtualenv__f01a7b__tldextract-1.2.3"
```

Comment:
Magic string `f01a7b`?

Reply:
it's a md5 hash of sys.prefix.  not sure a better way?